### ISSUE TITLE
Data Organization

### ISSUE THREAD

**Issue by lcaronson** [acrarshin/RPNet#4]
Hello, 

Can anyone provide insight on how to organize the data? I have downloaded the annotated ECG signals, but I am unsure how to organize the data folder to avoid errors.

data_path = "original_data/data"

 I am also unsure what the below path is referring to.

reference_path = "original_data/ref"

Any help would be greatly appreciated!

**Comment by acrarshin** [comment]
As mentioned in the paper, the data was trained on CPSC. This dataset does not have any complex preprocessing method. It is already segmented into windows of length 5 minutes along with a reference rpeak annotation. 

For evaluation on MITDBIH, MITST and NSTDB, the data is given in wfdb format, The preprocessing is done in utils.py in function obtain_data(). 

Hope this cleared it up.

**Comment by lcaronson** [comment]
Okay, so I downloaded the data from CPSC and placed those .mat files in my data_path. I think that is all set. 

But what goes in the reference_path folder? I downloaded the REFERENCE.csv file from CPSC, but I am struggling to understand how I place that in the reference_path folder without getting errors.

Thanks a lot!

**Comment by karaposu** [comment]
> I downloaded the data from CPSC

I can't download the data. It says page not found.  When did you download the data? Can you share it with me?

This is the page I tried to download:
http://www.icbeb.org/file/train.rar

**Comment by lcaronson** [comment]
Yes, this is the root of the issue, but I have found the correct training dataset. Follow the link below and scroll down the page to the lines I have quoted below. There you will find the correct training dataset. Hope this helps!

http://2019.icbeb.org/Challenge.html

"[2019-6-21] To solve the confusion caused by the old version of evaluation algorithm while dealing with the boundary problem, we fixed and resubmit all scoring code. Meanwhile, some labeling error in training set was fixed and resubmit as [Training set](http://2019.icbeb.org/file/train.rar)!"
